src/containers/BooksList.js

```diff
@@ -14,26 +16,26 @@ const BooksList = ({
   const handleFilterChange = (event) => changeFilter(event.target.value);
   const filteredBooks = filter === 'ALL' ? books : books.filter((item) => item.category === filter);
   const data = filteredBooks.map((item) => (
-    <Book
-      key={item.id}
-      book={item}
-      deleteBook={() => handleRemoveBook(item)}
-    />
+    <Book key={item.id} book={item} deleteBook={() => handleRemoveBook(item)} />
   ));
   return (
     <>
-      <CategoryFilter handleFilterChange={handleFilterChange} />
-      <table className="books-table">
-        <thead>
-          <tr>
-            <th>ID</th>
-            <th>Title</th>
-            <th>Category</th>
-            <th>Action</th>
-          </tr>
-        </thead>
-        <tbody>{data}</tbody>
-      </table>
+      <div className="books-section">
+        <div className="navBar">
+          <span id="main-logo">Bookstore CMS</span>
+          <p className="books">BOOKS</p>
+          <CategoryFilter
+            handleFilterChange={handleFilterChange}
+            className="categoryDropdown"
+          />
+          <div className="logoSec">
+            <img src={profileImg} alt="logo" className="logo_1" />
```

Comment:
- [ ] Please align this profile image to the left with the rest of the content, it was already there but after the changes that you did, it is now too much to the right, everything else looks awesome now, great job!

![image](https://user-images.githubusercontent.com/38522578/135939347-7f2a51a7-1ea8-4ad1-b349-5444226fba27.png)

Reply:
Done